lib/Notes/NoteViewPage/components/NoteView/NoteView.js

```diff
@@ -105,12 +106,14 @@ export default class NoteView extends Component {
 
   renderReferredRecord() {
     const {
+      assignedContent,
       referredEntityData,
       entityTypeTranslationKeys,
     } = this.props;
 
     return (
       <ReferredRecord
+        assignedContent={assignedContent}
```

Comment:
I don't think we should render `assignedContent` inside `ReferredRecord`. Referred record renders default data of the referred entity, `assignedContent` (`renderReferredRecordData` is a better name, I guees) allows us to choose what data should be rendered and to format this data properly. `ReferredRecord` and `renderReferredRecordData` are equivalent, so it would be better to have something like `props.renderReferredRecordData() || <ReferredRecord />`